<launch>
<arg name="Imu_name" default="um7" />

  <include file="$(find um7)/launch/um7_pub_and_display.launch"> </include>

  <node pkg="imu_calibration" type="imu_acc_gyro_calibration.py" name="$(arg Imu_name)_imu_acc_gyro_calibration" output="screen">
	<param name="imu_data_topic_name" value="/sensor/imu/um7/data" />
	<param name="mag_data_topic_name" value="/sensor/imu/um7/magfield_msg" />
    <param name="imu_name" value="CHRobotics_imu" />
    <param name="package_path" value="$(find imu_calibration)/configs/" /> -->
	<!-- <param name="mag_data_vector_topic_name" value="/sensor/imu/um7/magfield" /> -->
  </node>

</launch>
